cmd/crane/cmd/index.go

```diff
@@ -229,6 +260,7 @@ The platform for appended manifests is inferred from the config file or omitted
 	cmd.Flags().StringVarP(&newTag, "tag", "t", "", "Tag to apply to resulting image")
 	cmd.Flags().StringSliceVarP(&newManifests, "manifest", "m", []string{}, "References to manifests to append to the base index")
 	cmd.Flags().BoolVar(&dockerEmptyBase, "docker-empty-base", false, "If true, empty base index will have Docker media types instead of OCI")
+	cmd.Flags().BoolVar(&flatten, "flatten", false, "If true, appending an index will append each of its children rather than the index itself")
```

Comment:
```suggestion
	cmd.Flags().BoolVar(&flatten, "flatten", true, "If true, appending an index will append each of its children rather than the index itself")
```

because 

> it's more likely that they would want to merge them

Or maybe this means we want a `crane index concat` or something?

Reply:
Done, but I can't believe you would willfully break all our existing users.